site/pages/rest-api/examples/update-site.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Update Site
+---
+
+Updates an existing site on the server with the contents of `site.xml`.
+
+You can get the list of sites with the following commands:
+
+# cURL
```

Comment:
Changes made. Created "Get Sites" page and changed "Update Sites" to reference it.